docs/src/postgresql_conf.md

```diff
@@ -180,8 +180,8 @@ network disruptions. For more details, refer to the
 
 The operator requires PostgreSQL to output its log in CSV format, and the
 instance manager automatically parses it and outputs it in JSON format.
-For this reason, all log settings in PostgreSQL are fixed and cannot be
-changed.
+For this reason, some log settings in PostgreSQL, as shown in 
+[this section](#fixed-parameters), are fixed and cannot be changed.
```

Comment:
IMO this can be simplified. The subordinated clause "as shown", could go at the end.
I know this was in the original, but "fixed and cannot be changed" is redundant.
Also, if "some" parameters are fixed, might be nice to show some that are not.